docs/source/guide/get_started/introduction.rst

```diff
@@ -0,0 +1,27 @@
+Introduction
```

Comment:
I prefer to move it to the main index.rst before the table of content + add logo -> it will be the first page that users will see